android/src/main/kotlin/dev/rexios/polar/PolarPlugin.kt

```diff
@@ -124,6 +123,13 @@ class PolarPlugin :
         call: MethodCall,
         result: Result,
     ) {
+        if (wrapperInternal == null) {
+            try {
```

Comment:
Is the try/catch necessary? Were you getting exceptions here?

Reply:
No I added it in since their example in the SDK repo had a try catch around it